projects/v3/src/app/services/hubspot.service.ts

```diff
@@ -130,7 +130,7 @@ export class HubspotService {
       submitParam.fields.push(
         {
           name: "TICKET.team",
-          value: this.storage.getUser().teamName
+          value: this.storage.getUser().teamName ? this.storage.getUser().teamName : ''
```

Comment:
@sasangachathumal just to note, a better approach 
to this purposr is to assign this `this.storage.getUser().teamName` value to a variable and reuse it.